website/integrations/services/wazuh/index.md

```diff
@@ -0,0 +1,224 @@
+      config:
+        idp:
+          metadata_file: "/etc/wazuh-indexer/opensearch-security/idp-metadata.xml"
+          entity_id: "wazuh-saml"
+        sp:
+          entity_id: "wazuh-saml"
+        kibana_url: "https://<YOUR_WAZUH_IP_OR_HOSTNAME>"
+        roles_key: Roles
+        exchange_key: "MIIGIODCCA+ygAwIBAgITREIpdNn++usuy4Gq5g0W2/SQs..."
+     authentication_backend:
+       type: noop
+```
+
+### Step 6 - securityadmin.sh
+
+After you save the `config.yml` you need to restart the wazuh manager and tell it to load the modified file - this can be done automagically using the `securityadmin.sh` wazuh provides with the following command:
+
+```bash
+export JAVA_HOME=/usr/share/wazuh-indexer/jdk/ && bash /usr/share/wazuh-indexer/plugins/opensearch-security/tools/securityadmin.sh -f /etc/wazuh-indexer/opensearch-security/config.yml -icl -key /etc/wazuh-indexer/certs/admin-key.pem -cert /etc/wazuh-indexer/certs/admin.pem -cacert /etc/wazuh-indexer/certs/root-ca.pem -h localhost -nhnv
+```
+
+If all goes well this shows `Done with success` in the end
+
+### Step 7 - roles_mapping.yml
+
+In order to map the backend roles from authentik to wazuh specific roles you need to adjust the `/etc/wazuh-indexer/opensearch-security/roles_mapping.yml`
+
+Open the file and scroll to the following section:
+![Alt text](roles_mapping.png)
+
+under `backend_roles` add `wazuh-admin` and check that reserved is `false` + the rest looks like below:
+
+```yml
+all_access:
+    reserved: false
+    hidden: false
+    backend_roles:
+        - "wazuh-admin"
+        - "admin"
+    hosts: []
+    users: []
+    and_backend_roles: []
+    description: "Maps admin to all_access"
+```
+
+save the file and use the securityadmin.sh with the following command to load the adjusted `roles_mapping.yml`:
+
+```bash
+export JAVA_HOME=/usr/share/wazuh-indexer/jdk/ && bash /usr/share/wazuh-indexer/plugins/opensearch-security/tools/securityadmin.sh -f /etc/wazuh-indexer/opensearch-security/roles_mapping.yml -icl -key /etc/wazuh-indexer/certs/admin-key.pem -cert /etc/wazuh-indexer/certs/admin.pem -cacert /etc/wazuh-indexer/certs/root-ca.pem -h localhost -nhnv
+```
+
+If all goes well this shows `Done with success` in the end
+
+### Step 8 - wazuh.yml
+
+Check `/usr/share/wazuh-dashboard/data/wazuh/config/wazuh.yml` and make sure that `run_as` is set to `false`.
+![](wazuhyml.png)
+
+### Step 9 - security role mapping
+
+open the wazuh dashboard - click on the downward pointing triangle next to the wazuh logo then on `Security` and `Roles mapping`.
+![](roles_mapping1.png)
+We will now add a new role mapping - add any name + the respective Roles -> in this case administrator and add a new custom rule at the bottom that matches (`FIND`) the `user_name` to `wazuh-admin`.
+![](saml-admin.png)
+
+### Step 10 - final step - opensearch_dashboards.yml
+
+The last step is to adapt the `/etc/wazuh-dashboard/opensearch_dashboards.yml` and add three lines to the bottom of the file:
```

Comment:
```suggestion
The last step is to modify `/etc/wazuh-dashboard/opensearch_dashboards.yml` and add three lines to the bottom of the file:
```

Reply:
"adapt" would work as a verb, if you prefer it; we usually say wither "edit" or "modify" when we write about files.